### PR TITLE
Adds "law 666" for cyborg devils.

### LINKED ISSUE #19627: Cyborgs can be devil agents
Ran a round of devil agents and a roundstart borg was picked as a devil.
@lordpidey

### PR DESCRIPTION
**Pull request by lordpidey**
fixes #19627

Well, more like fleshes it out and turns it into a feature.

### CODE REVIEW

**Comment by Bawhoppen** [comment]
should have waited for 20 more prs so this would be #19666

File: code/modules/mob/living/silicon/silicon.dm [hunk 1]
```diff
@@ -230,6 +231,14 @@
 //		src << text ("Switching Law [L]'s report status to []", lawcheck[L+1])
 		checklaws()
 
+	if (href_list["lawdevil"]) // Toggling whether or not a law gets stated by the State Laws verb --NeoFite
+		var/L = text2num(href_list["lawdevil"])
+		switch(devillawcheck[L])
+			if ("Yes") devillawcheck[L] = "No"
```
Comment (RemieRichards) [inline code/modules/mob/living/silicon/silicon.dm]: not on the same line.

File: code/datums/ai_laws.dm [hunk 2]
```diff
@@ -219,6 +223,10 @@
 
 /datum/ai_laws/proc/show_laws(who)
 
+	if (src.devillaws && src.devillaws.len) //Yes, devil laws go in FRONT of zeroth laws, as the devil must still obey it's ban/obligation.
```
Comment (RemieRichards) [inline code/datums/ai_laws.dm]: no ```src.```, stop.

**Comment by GunHog** [comment]
Does this work on AIs too, or just borgs?

**Comment by optimumtact** [comment]
code/datums/mind.dm:1197:error: iscyborg: undefined proc

**Comment by lordpidey** [comment]
I've specifically excluded AIs from being devils, as they're immobile, which would render it very unfun, as people would need to visit your core to get a contract.

**Comment by killerx09** [comment]
That honestly sounds like a lot. You now have a pleatoria of eggborg salesman running around trying to sell you contracts, and when they accept you could be like

"Welcome. TO MY LAIR!"

**Pull request merged**